MAUI/Pull-to-Refresh/customization.md

```diff
@@ -0,0 +1,640 @@
+---
+layout: post
+title: Customization in .NET MAUI PullToRefresh control | Syncfusion
+description: Learn about Customization support in Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---
+
+# Customization in .NET MAUI PullToRefresh (SfPullToRefresh)
+
+The .NET MAUI PullToRefresh control supports customization of various features, including TransitionMode, PullingThreshold, ProgressBackground, ProgressColor, and more. The control can be personalized using the following properties.
+
+## PullableContent
+
+The `PullableContent` is the main view of the `PullToRefresh` control on which the desired items can be placed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="6 11" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh"
+                                PullingThreshold="120"
+                                RefreshContentHeight="30"
+                                RefreshContentThreshold="30"
+                                RefreshContentWidth="30">
+        <syncfusion:SfPullToRefresh.PullableContent>
+                <Label x:Name="Monthlabel" 
+                        TextColor="White" 
+                        HorizontalTextAlignment="Center"   
+                        VerticalTextAlignment="Start" />
+        </syncfusion:SfPullToRefresh.PullableContent>
+    </syncfusion:SfPullToRefresh>
+
+{% endhighlight %}
+{% endtabs %}
+
+## TransitionMode
+
+The `TransitionMode` property specifies the mode of the animations. It has the following two modes:
+
+* `SlideOnTop`
+* `Push`
+
+The default transition is `SlideOnTop` that draws the RefreshView on top of the `PullableContent`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                TransitionMode="SlideOnTop" />
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    pullToRefresh.TransitionMode = PullToRefreshTransitionType.SlideOnTop;
+
+{% endhighlight %}
+{% endtabs %}
+
+![PullToRefresh with slide on top transition mode](Images/customization/maui-pulltorefresh-getting-started.png)
```

Comment:
follow standard for Alt text
![Slide-on-top transition mode in .NET MAUI PullToRefresh] 
(or)
![.NET MAUI PullToRefresh with slide-on-top transition mode]

check the whole document for this comment.